### ISSUE TITLE
procinfo: incorrect handling of abstract domain sockets

### ISSUE THREAD

**Issue by disconnect3d** [pwndbg/pwndbg#1544]
The `procinfo` command reads some of the procfs files and then it may `.decode()` them to utf-8:
https://github.com/pwndbg/pwndbg/blob/9522a94baabb4d9a4fda2928e1f1754b477a6fb6/pwndbg/gdblib/net.py#L5-L24

However, the /proc/net/unix file may contain bytes that can't be decoded to utf-8. I believe those are related to abstract UDS (Unix Domain Sockets): https://unix.stackexchange.com/questions/206386/what-does-the-symbol-denote-in-the-beginning-of-a-unix-domain-socket-path-in-l

```
pwndbg> pi pwndbg.gdblib.file.get("/proc/net/unix").decode()
Python Exception <class 'UnicodeDecodeError'> 'utf-8' codec can't decode byte 0x9e in position 7126: invalid start byte:
Error while executing Python code.

pwndbg> pi
>>> open("/proc/net/unix", "rb").read().decode()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
UnicodeDecodeError: 'utf-8' codec can't decode byte 0x9e in position 7126: invalid start byte
>>> open("/proc/net/unix", "rb").read()[:7250][-300:]
b'a/nmbd/unexpected\n0000000000000000: 00000002 00000000 00010000 0001 01 25365 /var/run/libvirt/libvirt-admin-sock\n0000000000000000: 00000002 00000000 00000000 0002 01 23302 @@@@\x9e\x05@@\x01=\r@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@\n0000000000000000: 0'
```

**Comment by disconnect3d** [comment]
Also this bug may not always be reproducible directly via `procinfo`; I am not completely sure what conditions need to happen for it, maybe we need a socket in our app to be opened first?
```
pwndbg> procinfo
exe        '/bin/ls'
pid        27150
tid        27150
ppid       27138
uid        [1000, 1000, 1000, 1000]
gid        [1000, 1000, 1000, 1000]
groups     [4, 24, 27, 30, 33, 46, 108, 113, 114, 120, 124, 1000]
fd[0]      /dev/pts/3
fd[1]      /dev/pts/3
fd[2]      /dev/pts/3

pwndbg> pi pwndbg.gdblib.file.get("/proc/net/unix").decode()
Python Exception <class 'UnicodeDecodeError'> 'utf-8' codec can't decode byte 0x9e in position 7126: invalid start byte:
Error while executing Python code.
```

**Comment by disconnect3d** [comment]
This should be fixed by #1545
